docs/how-to-guides/construct-test-data.md

```diff
@@ -0,0 +1,96 @@
+# How to construct test data based on documents
+This guide will instruct you on how to generate test data for RAG systems utilizing existing documents.
+Previously evaluating the performance of RAG systems required the creation of test data. This could be done manually, a process that required significant time and effort, or by purchasing pre-made test data, which could be costly.
+
+This test data generation process streamlines this by leveraging the capabilities of llm to automatically generate the test data. This not only reduces the effort required but also eliminates the need for additional expenditures.
+
+By following this guide, you will learn how to:
+1. Customize the test data generation process by tuning flow prompts.
+2. Generate high-quality test data quickly and easily by running a test data generation script.
+
```

Comment:
## Prerequisites

1. Prepare documents.
<document requirements>
2. Install required packages.
<run in local requirements>
<run in cloud requirements>
<specific document type requirements>